app/controllers/api/v1/instructor_students_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::InstructorStudentsController < ApplicationController
+  def index
+    return json_response({data: "Invalid Query Parameters"}, 400) unless params.has_key?(:mod)
```

Comment:
May be worth if down the line for us to abstract the error handling to the application controller so all controllers can use it.